### PR TITLE
expose anzu--update-timer to the user

### PR DESCRIPTION
**Pull request by LemonBreezes**
Hello. This PR exposes `anzu--update-timer' to the user so that I can call its lambda manually from Elisp. 

The idea is that I want to use Corfu to complete search queries in the minibuffer and to do so, I must be able to manually update the visual hint because completion & `minibuffer-exit` happen simultaneously (no time for an idle timer).
image

### CODE REVIEW

**Comment by tarsius** [comment]
I was asked to review this by mail, by @LemonBreezes.

@gonewest818, you volunteered as the new maintainer of this package. Are you still interested in this position? If so, then please have a look at this pull-request.

@LemonBreezes, if we don't hear from @gonewest818 in a week or two, then please ping me and I will do the review then. Sorry for the delay, and thanks for your patience!

**Comment by LemonBreezes** [comment]
> I was asked to review this by mail, by @LemonBreezes.
> 
> @gonewest818, you volunteered as the new maintainer of this package. Are you still interested in this position? If so, then please have a look at this pull-request.
> 
> @LemonBreezes, if we don't hear from @gonewest818 in a week or two, then please ping me and I will do the review then. Sorry for the delay, and thanks for your patience!

Thanks for the response, @tarsius. Sure!

**Comment by tarsius** [comment]
I've [read](https://www.reddit.com/r/emacs/comments/1afr34c/i_love_magit_forges/koh9q55/) that #246 makes this package usable again.

File: anzu.el [hunk 1]
```diff
@@ -508,11 +509,11 @@
   (let ((curbuf (current-buffer))
         (blink-matching-paren nil)
         (anzu--history (anzu--transform-from-to-history))
-        timer is-input)
+        is-input)
     (unwind-protect
         (minibuffer-with-setup-hook
             #'(lambda ()
-                (setq timer (run-with-idle-timer
+                (setq anzu--update-timer (run-with-idle-timer
```
Comment (tarsius) [inline anzu.el:516]: Please reindent below.

File: anzu.el [hunk 2]
```diff
@@ -645,7 +646,7 @@
...
```
Comment (tarsius) [inline anzu.el:649]: Here too.

**Review by tarsius — CHANGES_REQUESTED** [review]

**Comment by tarsius** [comment]
> By the way, if you want, I can maintain this package.

I've given you commit access. Thanks for volunteering!

Please review the other pull-request too. It looks okay to me.

**Comment by LemonBreezes** [comment]
> > By the way, if you want, I can maintain this package.
> 
> I've given you commit access. Thanks for volunteering!
> 
> Please review the other pull-request too. It looks okay to me.

Sure. Also, I just resolved the indentation issue.

**Review by tarsius — APPROVED** [review]

**Pull request merged**